server/public/styles.css

```diff
@@ -0,0 +1,25 @@
+.col-lg-3 {
+  display: flex;
+  flex-direction: row;
+  height: 65vh;
+}
+
+img {
+  object-fit: contain;
+}
+
+.card-body {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-top: 2rem;
+}
```

Comment:
Don't modify Bootstrap's classes. Instead, make your own custom classes. You don't want to confuse team members when these classes deviate in behavior from the Bootstrap docs.

Plus a number of these styles can be accomplished with Bootstrap's utility classes.

https://getbootstrap.com/docs/4.4/utilities/flex/

https://getbootstrap.com/docs/4.4/utilities/spacing/